airflow/jobs.py

```diff
@@ -1821,7 +1821,8 @@ def _execute(self):
 
             for ti in run.get_task_instances():
                 # all tasks part of the backfill are scheduled to run
-                ti.set_state(State.SCHEDULED, session=session)
+                if ti.state == State.NONE:
```

Comment:
Just to make sure we are clear here: we will not be exactly the same as 1.7.1. If we find out that we need this we can do a command line option (I'll create a lira for that anyway).